src/pages/workspaces/workspace/Analyses.js

```diff
@@ -145,73 +143,52 @@ const AnalysisCard = ({ namespace, name, updated, metadata, toolLabel, listView,
+    }
+  }, [getDisplayName(name)])
+
+  const toolIconSrc = Utils.switchCase(application,
+    [tools.Jupyter.label, () => jupyterLogo],
+    [tools.RStudio.label, () => rLogo])
+  const toolIcon = div({ style: { marginRight: '1rem' } }, [
+    img({ src: toolIconSrc, style: { height: 40, width: 50 } })
+  ])
 
-  const appIconSrc = Utils.switchCase(toolLabel, [tools.Jupyter.label, () => jupyterLogo], [tools.RStudio.label, () => rLogo])
-  const appIcon = div({ style: { marginRight: '1rem' } }, [
-    img({ src: appIconSrc, style: { height: 40, width: 40 } })
+  const toolContainer = div({ style: { display: 'flex', flex: 1, flexDirection: 'row', alignItems: 'center' } }, [
+    toolIcon,
+    // this is the tool name, i.e. 'Jupyter'. It is named identical to the header row to simplify the sorting code at the cost of naming consistency.
+    application
   ])
 
   return a({
     href: analysisLink,
-    style: {
-      ...Style.elements.card.container,
-      ...analysisCardCommonStyles(listView),
-      flexShrink: 0
-    }
-  }, listView ? [
-    appIcon,
-    toolLabel,
-    title,
-    div({ style: { flexGrow: 1 } }),
-    locked && h(Clickable, {
-      style: { display: 'flex', paddingRight: '1rem', color: colors.dark(0.75) },
-      tooltip: `This analysis is currently being edited by ${lockedBy || 'another user'}`
-    }, [icon('lock')]),
-    h(TooltipTrigger, { content: Utils.makeCompleteDate(updated) }, [
-      div({ style: { fontSize: '0.8rem', marginRight: '0.5rem' } },
-        `Last edited: ${Utils.makePrettyDate(updated)}`)
-    ]),
-    analysisMenu
-  ] : [
-    div({ style: { display: 'flex' } }, [
-      title,
-      div({ style: { flexGrow: 1 } }),
-      locked && h(Clickable, {
-        style: { display: 'flex', padding: '1rem', color: colors.dark(0.75) },
-        tooltip: `This analysis is currently being edited by ${lockedBy || 'another user'}`
-      }, [icon('lock')])
-    ]),
-    div({
-      style: {
-        display: 'flex',
-        alignItems: 'center',
-        justifyContent: 'space-between',
-        borderTop: `solid 1px ${colors.dark(0.4)}`,
-        paddingLeft: '0.5rem', paddingRight: '0.5rem', height: '2.5rem',
-        backgroundColor: colors.light(0.4),
-        borderRadius: '0 0 5px 5px'
-      }
-    }, [
-      h(TooltipTrigger, { content: Utils.makeCompleteDate(updated) }, [
-        div({ style: { fontSize: '0.8rem', marginRight: '0.5rem' } }, [
-          'Last edited: ',
-          Utils.makePrettyDate(updated)
+    style: _.merge({
+      ...Style.cardList.longCardShadowless
+    }, { marginBottom: '.75rem', paddingLeft: '1.5rem' })
+  }, [
+    toolContainer,
+    artifactName,
+    //The 10% allows for the longest date possible (September XX, XXXX), plus a locked symbol, without text wrapping when the screen is maximized
+    div({ style: { ...endColumnFlex, flexDirection: 'row' } }, [
+      div({ style: { flex: 1, display: 'flex' } }, [
```

Comment:
```suggestion
div({ style: { flex: 1, display: 'flex', justifyContent: 'flex-end' } }, [
```
This will simplify the code and the rendered markup without changing the end result of the display

Reply:
this doesn't seem to quite work... when this is done, the content in the table does not line up with its header. The current styling does that, while ensuring two parts (the date + menu) are positioned correctly relative to each other regardless of the browser sizing.